### ISSUE TITLE
Typing Lag

### ISSUE THREAD

**Issue by Porco24** [mugiwara85/CodeblockCustomizer#115]
When I enable the plugin, I occasionally experience input lag when typing, especially when inputting Chinese. However, the issue disappears after I disable the plugin.

In the video, I tested the problem as it occurred. I disabled all plugins except for CodeblockCustomizer, and the input lag persisted. The lag was alleviated after I disabled CodeblockCustomizer.

https://imgur.com/a/AM5NJfT

**Comment by mugiwara85** [comment]
I see now. So, it also occurs when typing outside of a code block. Is this a large document? Does it contain any code blocks?

**Comment by Porco24** [comment]
@mugiwara85 
This is a relatively large file, approximately 5500 characters in length. It was created using Templater and Periodic Notes as a weekly journal. The content mainly consists of document attribute, file references, headings, and regular text input, without any code blocks.

**Comment by Porco24** [comment]
So far, the issue hasn't occurred yet. It typically happens with large files and is sporadic in nature.

**Comment by mugiwara85** [comment]
This issue is probably related to how I parse the syntax tree, and the explorer issue is probably just CSS. I just need to find which one.

**Comment by Porco24** [comment]
Sorry, I didn't quite understand what "Moving Heading" means. The issue I'm experiencing is the same as the one shown in the issue video—it occurs when I try to move a file's location.

**Comment by bunsyy** [comment]
Here, I am having the same typing lag issue.

**Comment by felixsdys** [comment]
Yes, I'm also experiencing severe lagging when typing on a page with a lot of Wikilinks

**Comment by mugiwara85** [comment]
@Porco24 @bunsyy @felixsdys Sorry for the late reply. Do you all have this lag problem when `Enable Links usage` is enabled? Or is it not depending on this option?

**Comment by Porco24** [comment]
> [@Porco24](https://github.com/Porco24) [@bunsyy](https://github.com/bunsyy) [@felixsdys](https://github.com/felixsdys) Sorry for the late reply. Do you all have this lag problem when `Enable Links usage` is enabled? Or is it not depending on this option?

You mean "Enable editor active line highlight"?
I close this plugin sometime, and I update obsidna to v1.8.4, I can't reproduce the error

**Comment by mugiwara85** [comment]
@Porco24 @bunsyy @felixsdys Could you please download the `main.js` and `styles.css` from the repo, and overwrite yours with it, and test if the problem still persists? Please give me a feedback, if it is ok now. Thanks!

**Comment by mugiwara85** [comment]
Fixed in version 1.3.0
